src/log4net.Tests/Appender/AdoNet/Log4NetCommand.cs

```diff
@@ -24,7 +24,7 @@
 
 namespace log4net.Tests.Appender.AdoNet;
 
-public class Log4NetCommand : IDbCommand
+internal sealed class Log4NetCommand : IDbCommand
```

Comment:
Just a curiosity: isn't this a breaking change?
At least it would be a breaking change in Java.

Reply:
Yes, but this is in our test assembly and we don't ship it ;-)
This class should better be named MockDbCommand - I will change it in my next PR.